vmdb/app/models/ems_event/parsers/kubernetes.rb

```diff
@@ -0,0 +1,21 @@
+module EmsEvent::Parsers::Kubernetes
+  def self.event_to_hash(event, ems_id = nil)
+    log_header = "MIQ(#{name}.event_to_hash) ems_id: [#{ems_id}]"
+
+    $log.debug("#{log_header} event: [#{event.inspect}]")
+
+    event_type = "#{event[:kind].upcase}_#{event[:reason].upcase}"
```

Comment:
We don't currently do anything to distinguish the different events between the providers, so adding something to the `event_type` for Kubernetes would definitely be a deviation from the rest.  That's a fair point.

Instead, how about just adding a comment to the [event_handling.tmpl.yml](https://github.com/ManageIQ/manageiq/pull/2478/files#diff-7ee06aa36de058d7fabf829de5136b1bR1403) file to say that those specific events are Kubernetes events.  That seems like a decent compromise until we introduce better event handling and mapping (something that's been long in the works).